src/@carbon-info/views/Build/Build.tsx

```diff
@@ -40,8 +40,8 @@ const Build: React.FC = () => {
             </Grid>
             <Grid item xs={12} sm={7} md={4} className={inView && !isMobile ? "animate__animated animate__fadeInUp" : ""}>
               <CardWithCTA
-                title={"Propose a partnership"}
-                description={"Working towards a freer financial system"}
+                title={"Launch on Carbon EVM"}
```

Comment:
Update the previous Card, change title to "Develop with Carbon Core" and update the link